site/gdocs/Chart.tsx

```diff
@@ -23,6 +24,8 @@ export default function Chart({
         }
     }
 
+    const isExplorer = d.url.includes(`/${EXPLORERS_ROUTE_FOLDER}/`)
```

Comment:
there is a test in the Url package for this that it might makes sense to reuse:
https://github.com/owid/owid-grapher/blob/d43d1444209260707bdcb2e72ab478a0c0b82f15/packages/@ourworldindata/utils/src/urls/Url.ts#L133-L135